src/models/goal.js

```diff
@@ -93,6 +101,16 @@ export default (sequelize, DataTypes) => {
     previousStatus: {
       type: DataTypes.TEXT,
     },
+    mergedIntoId: {
```

Comment:
I know this is for merging goals, but in effect it directly correlates to the mapsTo convention we have been using recently. Personally I would prefer that naming convention.

Reply:
makes sense, will change